### PR TITLE
fix: Add debugging information for sourcemapper

### LINKED ISSUE #932: Could not determine the output file associated with the transpiled input file
### Problem you have encountered:
Receiving "Could not determine the output file associated with the transpiled input file" error when we deploy node js code.

### Configurations:
- typescript - v4.0.3
- node - v14.15.0
- webpack - v5.4.0

We tried to enable sourcemap with below webpack sourcemapers

- 'source-map'
- 'inline-source-map'
- 'hidden-source-map'
But we are not able to set debugger in typescript based project.

### What you expected to happen:
The debugger should be attached to a node js based project which contains code in TS and transpile through webpack.

### PR DESCRIPTION
**Pull request by Louis-Ye**
This PR is to add debugging information to source mapper so that it is easier to debug related issues from users.

Fixes #932

### CODE REVIEW

File: test/test-sourcemapper.ts [hunk 1]
```diff
@@ -52,16 +54,37 @@ function testTool(
...
+      async () => {
+        const start = Date.now();
+        sourcemapper = await sm.create([mapFilePath], logger);
+        assert(
+          Date.now() - start < QUICK_MILLISECONDS,
+          'should create the SourceMapper quickly'
+        );
+
+        // Verify if the debugging information is correctly printed.
+        assert.notStrictEqual(
```
Comment (mctavish) [inline test/test-sourcemapper.ts:71]: Could be just me, but I find the notStrictEqual(...indexOf(...), -1) to be hard to read.  Maybe matches instead?

File: test/test-sourcemapper.ts [hunk 2]
```diff
@@ -52,16 +54,37 @@ function testTool(
...
+      'for tool ' + tool + ' sourcemapper should be created correctly',
+      async () => {
+        const start = Date.now();
+        sourcemapper = await sm.create([mapFilePath], logger);
+        assert(
+          Date.now() - start < QUICK_MILLISECONDS,
+          'should create the SourceMapper quickly'
+        );
+
+        // Verify if the debugging information is correctly printed.
```
Comment (mctavish) [inline test/test-sourcemapper.ts:70]: Split this into a different test, and please provide an example of what the output is.  It's hard for me to parse it based off of the assertions below.

File: test/test-sourcemapper.ts (hunk 1 above)
Comment (Louis-Ye) [reply test/test-sourcemapper.ts]: Done.

File: test/test-sourcemapper.ts (hunk 2 above)
Comment (Louis-Ye) [reply test/test-sourcemapper.ts]: Done.

**Review by mctavish — APPROVED** [review]

**Pull request merged**